using-timescaledb/limitations.md

```diff
@@ -0,0 +1,42 @@
+# Limitations
+
+While TimescaleDB generally offers capabilities that go beyond what
+PostgreSQL offers, there are some limitations to using hypertables,
+and, in particular, distributed hypertables. This section documents
+the common limitations when using both regular and distributed
+hypertables.
+
+## Hypertable Limitations
+
+- Foreign key constraints referencing a hypertable are not supported.
+- Time dimensions (columns) used for partitioning cannot have NULL
+  values.
+- Unique indexes must include all columns that are partitioning
+  dimensions.
+
+## Distributed Hypertable Limitations
+
+All the limitations of regular hypertables also apply to distributed
+hypertables. In addition, the following limitations apply specifically
+to distributed hypertables:
+
+
+- Background job scheduling is not supported.
+- Continuous aggregates are not supported.
+- Compressed hypertables are not supported.
+- Tablespaces cannot be attached to a distributed hypertable on the
+  access node. It is still possible attach tablespaces on data nodes.
+- Roles and permissions are assumed to be consistent across the nodes
+  of a distributed database, but consistency is not enforced.
+- Tables referenced by foreign key constrains in a distributed
+  hypertable must be present on all data nodes.
+- Joins on data nodes are not supported. Joining a distributed
+  hypertable with another table requires the other table to reside on
+  the access node. This also limits the performance of joins on
+  distributed hypertables.
+
+Note that these limitations concern usage from the access node. Some
+currently unsupported features (like background scheduling and
+continuous aggregates) might still work on individual data nodes, but
+such usage is neither tested nor officially supported. Many of these
+limitations might be removed in future versions of TimescaleDB.
```

Comment:
I try to avoid making promises, even implicit ones like this. They look innocent, but they always bite back.

Reply:
FWIW, the intention here is to give some idea to the user that not all of these limitations are inherent to the design.

Reply:
We will need to remove many of the limitations and thus I prefer to keep current formulation or even make it stronger.